front/src/app/projects/projects.component.ts

```diff
@@ -7,21 +8,28 @@ import { Component, OnInit } from '@angular/core';
 })
 export class ProjectsComponent implements OnInit {
 
+  columns = 3;
+  rows = '2:1.5';
+
   projects = [{
+    id: 1,
     name: 'Project 1',
-    tools: [{id: '123', tool: 'tool 1'},
-    {id: '213', tool: 'tool 2'},
-    {id: '321', tool: 'tool 3'}],
-    date: '01-01-2019',
+    tools: [{tool: 'tool 1', date: '01/01/2019 14:00', status: 'positive'},
+    {tool: 'tool 2', date: '01/01/2019 14:00', status: 'positive'},
+    {tool: 'tool 2', date: '01/01/2019 14:00', status: 'positive'},
+    {tool: 'tool 3', date: '01/01/2019 14:00', status: 'negative'},
+    {tool: 'tool 2', date: '01/01/2019 14:00', status: 'positive'},
+    {tool: 'tool 2', date: '01/01/2019 14:00', status: 'positive'},
+    {tool: 'tool 2', date: '01/01/2019 14:00', status: 'positive'}],
     log: 'test log',
     status: 'negative'
     },
     {
+      id: 2,
       name: 'Project 2',
-      tools: [{id: '123', tool: 'tool 1'},
-      {id: '213', tool: 'tool 2'},
-      {id: '321', tool: 'tool 3'}],
-      date: '01-01-2019',
+      tools: [{tool: 'tool 1', date: '01/01/2019 14:00', status: 'positive'},
+      {tool: 'tool 2', date: '01/01/2019 14:00', status: 'waiting'},
+      {tool: 'tool 3', date: '01/01/2019 14:00', status: 'positive'}],
       log: 'test log',
       status: 'waiting'
       }
```

Comment:
Można to spokojnie przerzucić do stanu projektów w ngxs jako wartość startowa, a w tym komponencie użyć selektora na stan projektów.